admin/views/results/view-results.ejs

```diff
@@ -0,0 +1,96 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+  </div>
+  <div class="column-one-third">
+    <%- partial('partials/_name_and_sign_out') %>
+  </div>
+</div>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+  <h1 class="heading-xlarge"><%= pageTitle %></h1>
+  <p class="lede" id="lead-paragraph">
+    Until verified, results are subject to review. All results are out of 25.
+  </p>
+  <div>
+    <dl class="score-presenter">
+      <dd>
+        <h3>School Average</h3>
+        <div class="font-xlarge"><%= schoolScore %></div>
+      </dd>
+      <dd>
+        <h3>National Average</h3>
+        <div class="font-xlarge"><%= nationalScore ? nationalScore: '0.00' %></div>
```

Comment:
If the national score is not provided, then a placeholder such as 'N/A' is more appropriate, rather than providing a numeric answer that appears to be the result of a genuine calculation.

Reply:
@jon-shipley was thinking the same, updated.